javascript/javascript2/week1/intro-recipes.js

```diff
@@ -0,0 +1,15 @@
+function insertSiteTitle() {
+    const siteTitle = document.createElement("header");
+    siteTitle.classList.add("site-header");
+    siteTitle.innerHTML = `
+      <a href="/" class="site-logo">
+        <img src="./fork&flavor.png" alt="Fork&Flavor Logo" style="height: 100px; vertical-align: middle; margin-right: 10px;">
+        <span>Fork&amp;Flavor</span>
+      </a>
+    `;
+    document.body.prepend(siteTitle);
+  }
+  
+  window.addEventListener("DOMContentLoaded", () => {
```

Comment:
You can simplify this part by removing the arrow function
```suggestion
  window.addEventListener("DOMContentLoaded", insertSiteTitle)
```